README.md

```diff
@@ -1 +1,41 @@
+# Deprecation Changelog Generator
 
+This tool generates the deprecation notice section of a changelog for a Silverstripe CMS release.
+
+It also lets you know if you need to perform follow-up actions (e.g. if some removed API didn't have a deprecation notice).
+
+**This tool is only intended for use by Silverstripe core committers or the Silverstripe Ltd CMS Squad**
+
+## Setup
+
+1. Clone this repo somewhere.
+1. Run `composer install` to install its dependencies.
+1. Set up a directory somewhere that this tool will use for cloning Silverstripe CMS into and for outputting any long-form data.
```

Comment:
This really feels like it should be a `_data` style dir that's automatically created and .gitignored, similar to module-standardiser, rather than something you manually created and pass in as an argument

Reply:
I prefer the flexibility to have the output go wherever the heck I want it. If you want it directly in this dir, you can just have this dir as your current working dir and omit the `--dir` optional flag.

I'll update this note to be less "make a new directory" and more "choose where you want to be doing this"